### PR TITLE
RC Drone ArnavK28

### PR DESCRIPTION
**Pull request by ArnavK28**
## Describe your project in 1-3 sentences

 I am building a remote controlled quadcopter which will eventually include cameras and could be used for videography purposes.

## Hack Club Slack handle
@ArnavK

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: RC Drone ArnavK28 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Conversion rate: 1$ = ₹81.33
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Quadcopter Drone kit  | https://www.flyrobo.in/quadcopter-drone-combo-with-apm-2.8-kit-for-beginner-motor-plus-esc-plus-propeller-plus-flight-controller-plus-frame-plus-tx-rx-plus-power-module-plus-belt-flyrobo?tracking=ads&gclid=Cj0KCQiAn4SeBhCwARIsANeF9DKbRABd5ImIbbZdCxOwUy8_-gBlC_kkBEUR3jbZJXvUHudJvdrhi0EaAsDBEALw_wcB | ₹15,499  |
+| Lipo Power Battery | https://www.flyrobo.in/build-power-lipo-battery-11.1v-5500mah-3s-30c?tracking=5b853f18ae9a2  | ₹3,989 |
+| Lipo Power Charger | https://www.flyrobo.in/imax-b3-lipo-balance-charger-for-2-3-cell-lipo-battery?tracking=5b853f18ae9a2  | ₹519 |
+| Total           |                                       | ₹20,007 | 
+                                                          | $246.01 |
```
Comment (ArnavK28) [inline RC Drone ArnavK28:28]: Total price converted to USD (includes shipping and taxes)

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! we're back on tuesday and will start reviewing PRs in the order that they came on. 

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

_(note: from a first glance of your project i'm wondering if you're modifying this drone in anyway to make it your own or just building from a kit?)_

our Jan 15 11:59pm ET deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

**Review by khrj — APPROVED** [review]

**Review by jianmin-chen — COMMENTED** [review]
Looks good to go!

**Review by jianmin-chen — APPROVED** [review]
A member of @hackclub/bank-ops should get in touch with you soon :)

**Review by jianmin-chen — APPROVED** [review]

**Comment by khrj** [comment]
(In response to your earlier question, yes it's a solo project, I'm already building #18)

**Review by MelSmith104 — COMMENTED** [review]
hi @ArnavK28! im so excited that you're going to be building your first drone! im dm'ing you on slack now to chat through the next steps for the grant!!

**Comment by MelSmith104** [comment]
hi @ArnavK28! can you please send me your slack id? the one listed does not show up when i search :) thank you!

**Comment by ArnavK28** [comment]
Hey, sorry for that. Here's my slack id-
@Arnav:U04KJFEKL64

**Comment by MelSmith104** [comment]
thank you! @ArnavK28 just dm'ed you :)

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**